app/models/piece.rb

```diff
@@ -95,8 +95,8 @@ def move_to!(new_x, new_y)
       self.update(:x_position => new_x, :y_position => new_y)
 
     # Else if a friendly piece exists there, do nothing
-    elsif friend_or_foe.color == friendly_color
-      return
+    #elsif friend_or_foe.color == friendly_color
+      #return
```

Comment:
@leamerluck I'm thinking lines 96-99 can be deleted since they're just commented-out old code.